opentracing-jdbc/opentracing-jdbc-autoconfigure/src/test/java/org/zalando/opentracing/jdbc/autoconfigure/OpenTracingJdbcAutoConfigurationTest.java

```diff
@@ -0,0 +1,50 @@
+package org.zalando.opentracing.jdbc.autoconfigure;
+
+import com.zaxxer.hikari.HikariDataSource;
+import io.opentracing.mock.MockTracer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import javax.sql.DataSource;
+
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.springframework.aop.support.AopUtils.isAopProxy;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.NONE;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = NONE)
+@EnableAutoConfiguration
+class OpenTracingJdbcAutoConfigurationTest {
+
+    @Configuration
+    static class TestConfiguration {
+
+        @Bean
+        public MockTracer tracer() {
+            return new MockTracer();
+        }
+
+    }
+
+    @Autowired
+    private HikariDataSource dataSource;
+
+    @Test
+    void proxiesDataSource() {
+        assertTrue(isAopProxy(dataSource));
+    }
+
+    @Test
+    void supportsDirectAccess() throws SQLException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/whiskeysierra/opentracing-toolbox/pullRequest?prid=4701844)